### PR TITLE
Spring Boot 3: Optimize docs

### PR DESCRIPTION
**Pull request by JoergSiebahn**

### CODE REVIEW

File: docs/web.md [hunk 1]
```diff
@@ -88,170 +76,164 @@ This security implementation lacks some features compared to [sda-dropwizard-com
...
+configured with [`opa.` configuration properties](#configuration).
 
-```java
-@Constraints
-public class MyConstraints extends AbstractConstraints {
+??? info "Requests to the Open Policy Agent"
+    Requests to the server are determined by the base URL and the policy package.
+    Given the default base URL `http://localhost:8181` and an example package of `com.my.service`,
+    the Open Policy Agent server will be requested for authorization decision at
+    `http://localhost:8181/v1/data/com/my/package`.
```
Comment (frederic-huebner) [inline docs/web.md]: Shouldn't it be `http://localhost:8181/v1/data/com/my/service` then?

File: docs/starter-web.md [hunk 2]
```diff
@@ -0,0 +1,586 @@
...
+
+## Async
+
+The default Spring [async](https://javadoc.io/doc/org.springframework/spring-context/latest/org/springframework/scheduling/annotation/Async.html) 
+task executor is autoconfigured to transfer the request attributes of the 
+current request to the **Thread** running the asynchronous method.
+
+## Jackson
+
+Enables feature that make a Spring Boot service compliant with the REST guide of SDA SE.
```
Comment (frederic-huebner) [inline docs/starter-web.md:428]:
```suggestion
Enables feature that makes a Spring Boot service compliant with the REST guide of SDA SE.
```

File: docs/starter-web.md [hunk 3]
```diff
@@ -0,0 +1,586 @@
...
+using Open Policy Agent.
+
+The authorization is done by the [Open Policy Agent](https://www.openpolicyagent.org/). It can be 
+configured with [`opa.` configuration properties](#configuration).
+
+??? info "Requests to the Open Policy Agent"
+    Requests to the server are determined by the base URL and the policy package.
+    Given the default base URL `http://localhost:8181` and an example package of `com.my.service`,
+    the Open Policy Agent server will be requested for authorization decision at
+    `http://localhost:8181/v1/data/com/my/package`.
```
Comment (frederic-huebner) [inline docs/starter-web.md:89]: Shouldn't it be `http://localhost:8181/v1/data/com/my/service` then?

File: docs/starter-mongodb.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+## Configuration properties
+* `spring.data.mongodb.uri` _string_
+  * Mongo database URI.
+  * Example: `mongodb://exampleUser:examplePassword@mongoHost:27017`
+  *
+  Format: `mongodb://[username:password@]host1[:port1][,...hostN[:portN]][/[defaultauthdb][?options]]`
+  * Connection String Options need to be added to the end of the URI e.g.
+    * `?ssl=true` to enable SSL
+    * `?retryWrites=false` to disable retryable writes for the connection.
+    * `?readPreference=secondardPreferred` In most situations, operations read from secondary
```
Comment (frederic-huebner) [inline docs/starter-mongodb.md:30]:
```suggestion
    * `?readPreference=secondaryPreferred` In most situations, operations read from secondary
```

File: docs/starter-kafka.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+| `sda.kafka.consumer.retry.initialBackOffInterval` _int_ | The initial backoff of the retry in milli seconds.                                                                                | `1000`             | `1500`                                          | `SDA_KAFKA_CONSUMER_RETRY_INITIAL_BACKOFF_INTERVALL` |
+| `sda.kafka.consumer.retry.maxBackOffInterval` _int_     | The max backoff interval  in milli seconds.                                                                                       | `4000`             | `5000`                                          | `SDA_KAFKA_CONSUMER_RETRY_MAX_BACKOFF_INTERVALL`     |
+| `sda.kafka.consumer.retry.backOffMultiplier` _int_      | The multiplier beginning with the initial backoff.                                                                                | `2`                | `1.5`                                           | `SDA_KAFKA_CONSUMER_RETRY_INITIAL_BACKOFF_INTERVALL` |
```
Comment (frederic-huebner) [inline docs/starter-kafka.md:18]: To which types are we referring here? Concerning the example this seems to be rather a decimal.

Comment (frederic-huebner) [inline docs/starter-kafka.md:18]: To which types are we referring here? Concerning the example this seems to be rather a decimal.

File: docs/starter-web.md (hunk 3 above)
Comment (JoergSiebahn) [reply docs/starter-web.md]: fixed

File: docs/starter-mongodb.md (hunk 4 above)
Comment (JoergSiebahn) [reply docs/starter-mongodb.md]: fixed

File: docs/starter-kafka.md (hunk 5 above)
Comment (JoergSiebahn) [reply docs/starter-kafka.md]: I changed the config to double

**Review by frederic-huebner — APPROVED** [review]

**Pull request merged**